doc/_extensions/external_content.py

```diff
@@ -86,8 +90,17 @@ def sync_contents(app: Sphinx) -> None:
         app: Sphinx application instance.
     """
 
-    srcdir = Path(app.srcdir)
+    srcdir = Path(app.srcdir).resolve()
     to_copy = []
+    to_delete = set({f for f in srcdir.glob("**/*") if not f.is_dir()})
+    to_keep = set(
+        {
```

Comment:
What's up with the redundant `set({ ... })` expressions? They can just be `set(...)`.

Reply:
oops, fixed thanks